openpype/hosts/zbrush/startup/startup.txt

```diff
@@ -0,0 +1,77 @@
+[VarDef, crlf, [StrMerge, [StrFromAsc, 13], [StrFromAsc, 10]]]
+[VarDef, quot, [StrFromAsc, 34]]
+[VarDef, temp, [FileNameResolvePath, "_deleteme"]]
+[VarDef, bat, [StrMerge, temp, ".bat"]]
+[VarDef, txt, [StrMerge, temp, ".txt"]]
+[VarSet, offset, 0]
+
```

Comment:
Zscript files seem to support comments with `//`

I feel like code in this file will heavily benefit some clear comments as to what it's doing and why. :)

Reply:
It is just a headache to read:( I will add some later.